components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/artifact/SAMLSSOSoapMessageService.java

```diff
@@ -55,6 +55,11 @@ public class SAMLSSOSoapMessageService {
     private static final String CONTENT_TYPE = "text/xml; charset=utf-8";
     private static final Log log = LogFactory.getLog(SAMLSSOSoapMessageService.class);
 
+    /**
+     * Default ssl protocol for client
+     */
+    private static final String protocol = "TLSv1.2";
```

Comment:
final variables should be in block capitals.. also let's change the variable name that is suggestive of TLSv1.2 rather than just "protocol"